lib/galaxy/datatypes/sniff.py

```diff
@@ -198,24 +198,34 @@ def convert_newlines_sep2tabs( fname, in_place=True, patt="\\s+", tmp_dir=None,
         return ( i + 1, temp_name )
 
 
-def get_headers( fname, sep, count=60, is_multi_byte=False ):
+def get_headers( fname, sep, count=60, is_multi_byte=False, commentDesignator=None ):
```

Comment:
It's really a nitpick, but can you use `comment_designator` for the parameter name instead? We don't use mixedCase.